cfgmgr/buffermgr.cpp

```diff
@@ -34,6 +34,8 @@ void BufferMgr::readPgProfileLookupFile(string file)
     ifstream infile(file);
     if (!infile.is_open())
     {
+        m_pgfile_readable = false;
```

Comment:
Do you think m_pgfile_processed a more suitable name for this variable? I would think setting it to false by default, and flip it to true once the file processing is done will serve your purpose better?

Reply:
The purpose was to check whether the "pg-lookup-file" was there or not, if not, we don't do the buffer settings. It renamed to cover the case where the file somehow not readable. 
By looking at the code, there wasn't any exist after this "is_open" check and before the entire function.  This means  if we had m_pgfile_processed defined, we would have always set it to true after this check. I,E, there was no difference between the current approach and if we had a new naming.
Anyway, I changed it so everybody is happy.